src/app/search/search.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-search',
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.scss']
+})
+export class SearchComponent implements OnInit {
+  
+  constructor() { }
+
+  ngOnInit() {
+  }
+  queryString:string = null;
```

Comment:
I recommend declaring fields before methods and even before the constructor. It makes code more readable.